src/style/AccountPage.less

```diff
@@ -15,12 +15,101 @@
 	}
 }
 
-#account {
-	.navbar {
-		.login-link {
-			color: #AD2016;
-			& a {
-				color: #AD2016;
+.AccountPage {
+
+	.user {
+		img {
+			width: 100%;
+			@media @md-max {
+				width: 25%;
+				margin-bottom: 1rem;
+			}
+		}
+		h1 {
```

Comment:
Redefining fonts for a specific section is usually a mistake, as it leads to inconsistent styling + harder to maintain css.

Can we remove this?
Should it be promoted to a universal site-wide rule?

If this is needed as a specific patch, please leave a brief comment for why.